src/containers/about/MeetTheTeam.js

```diff
@@ -105,12 +105,14 @@ const MeetTheTeam = () => (
           {
             TEAM_MEMBERS.map((teamMember :Object, index :number) => {
               if (index > -1 && index < 3) {
+                let founderText = 'Founder &';
+                if (index === 1 || index === 2) founderText = 'Co-Founder &';
```

Comment:
1. I  think `index < 3` should suffice.
2. Instead of hardcoding the index, perhaps you could consider setting some property to the `teamMember` Object and choosing what value to display based on that property?